### PR TITLE
Fix outdated links

### PR DESCRIPTION
**Pull request by guessi**
### Description

Some legacy Amazon EKS doc links need update as upstream doc changed location.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [X] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [X] Manually tested
- [X] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: pkg/awsapi/eks.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ type EKS interface {
 	// that is created for your cluster. In most cases, it takes several minutes to
 	// create a cluster. After you create an Amazon EKS cluster, you must configure
 	// your Kubernetes tooling to communicate with the API server and launch nodes into
-	// your cluster. For more information, see Managing Cluster Authentication (https://docs.aws.amazon.com/eks/latest/userguide/managing-auth.html)
+	// your cluster. For more information, see Managing Cluster Authentication (https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html)
```
Comment (Himangini) [inline pkg/awsapi/eks.go:47]: please remove the changes here, its an autogenerated file.

**Review by Himangini — COMMENTED** [review]
Thanks for updating the links, they all look good to me.
one small nit.

File: pkg/awsapi/eks.go (hunk 1 above)
Comment (guessi) [reply pkg/awsapi/eks.go]: Noticed it was fixed by https://github.com/eksctl-io/eksctl/pull/7312/commits/9c9f66fab6995677eae17f9b9b5e4a2eeddfd52b

PR updated.

**Comment by guessi** [comment]
Ummm... Unit test failed with unrelated file change 🤔

**Comment by guessi** [comment]
Any chance to get it merged?

**Review by yuxiang-zhang — APPROVED** [review]

**Pull request merged**